Assets/HoloToolkit/Input/Scripts/GamePad/XboxController/XboxControllerHandlerBase.cs

```diff
@@ -2,131 +2,187 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Collections;
 using UnityEngine;
 
 namespace HoloToolkit.Unity.InputModule
 {
-    public class XboxControllerHandlerBase : MonoBehaviour, IXboxControllerHandler
+    public class XboxControllerHandlerBase : GamePadHandlerBase, IXboxControllerHandler
```

Comment:
Was this supposed to be added to the InputManager, to replace GamepadInput.cs? I don't see it.

Reply:
`GamePadHandlerBase` itself wasn't, but the `XboxControllerHandlerBase` was and it inherits from this class

Reply:
Hmm, I don't see `XboxControllerHandlerBase` in the prefab, just `XboxControllerInputSource`.

Reply:
I'll double check.

Reply:
Fixed.